include/CL/opencl.hpp

```diff
@@ -4618,6 +4645,41 @@ class Image : public Memory
         }
         return param;
     }
+
+#if defined(cl_ext_image_requirements_info)
+    static cl_int getImageRequirementsInfoExt(const Context &context,
```

Comment:
Debatably this should be a member function of the `Context` similar to `getSupportedImageFormats`.

If we're going to keep it as a static function though I don't think we need "Image" in the name since it would be redundant: `Image::getImageRequirementsInfoExt`

Reply:
I felt that clGetSupportedImageFormats is rather more general information and clGetImageRequirementsInfoEXT api is more closer to Image clas,
But if you will have other idea, I can move it to Context.